srcs/juloo.keyboard2/KeyModifier.java

```diff
@@ -180,6 +145,88 @@ private static char handleAccentChar(char c, int flags)
           case 'o': return 'ₒ';
           default: return c;
         }
+      case KeyValue.FLAG_ACCENT_ARROWS:
+        switch ((flags & KeyValue.FLAG_SHIFT))
+        {
+        case KeyValue.FLAG_SHIFT:
```

Comment:
This is `if ((flags & KeyValue.FLAG_SHIFT) != 1)`.

Reply:
Removed the switch-case, also removed the extra Alt variation, now it's just shift variation using a simpler if logic